.gitlab-ci.yml

```diff
@@ -92,8 +92,9 @@ publish:
   script:
     - docker load -i image.tar
     - docker tag $SERVICE_IMAGE $DOCKER_REPOSITORY:$COMMIT_TAG
```

Comment:
Do we need to tag it also as master regardless of `$COMMIT_TAG` value?

Reply:
good point! I added both branch (should be master or <tag>) and latest docker tags